frameworks/person-escort-record/questions/medication-while-moving.yml

```diff
@@ -0,0 +1,31 @@
+type: radio
+question: Do they need to take any medication while moving?
+description: Medication while moving
+options:
+  -
+    label: "Yes"
+    value: "Yes"
+    followup:
+      - medication-carrier-while-moving
+    followup_comment:
```

Comment:
Does this actually work?! :/ 

I thought that it was either a followup_comment OR followup fields?

Reply:
Ah I always thought we could have both, where a question can require a comment, and have a dependent question as well. If that doesn't play well with the FE we can have this as a followup question as well

Reply:
talked in person but for posterity: we don't currently support it, can add it in the future. For now move to new question